ewcmd.py

```diff
@@ -1018,12 +1015,11 @@ def weather_txt(id_server):
 
 
 
-		targetmodel.die(cause = ewcfg.cause_cliff)
+		die_resp = targetmodel.die(cause = ewcfg.cause_cliff)
 		targetmodel.persist()
 		await ewrolemgr.updateRoles(client=cmd.client, member=target)
-		if deathreport != "":
-			sewerchannel = ewutils.get_channel(cmd.message.server, ewcfg.channel_sewers)
-			await ewutils.send_message(cmd.client, sewerchannel, deathreport)
+		if die_resp != ewutils.EwResponseContainer(id_server = cmd.message.server.id_server):
```

Comment:
this check is unnecessary and doesn't work. if a response container is empty, it just won't post anything on .post(), and you can't compare objects like that

Reply:
alright, was just mimicking what was there before, my bad, removed check